### PR TITLE
Add sentry probe

### PR DESCRIPTION
**Pull request by qdequele**
The probe will be available only on production if analytics has not been deactivated.

### CODE REVIEW

File: meilisearch-http/src/main.rs [hunk 1]
```diff
@@ -28,8 +28,21 @@ pub fn main() -> Result<(), MainError> {
...
+            if !opt.no_analytics {
+                let _sentry = sentry::init((
+                    "https://5ddfa22b95f241198be2271aaf028653@sentry.io/3060337",
+                    sentry::ClientOptions {
+                        release: sentry::release_name!(),
+                        ..Default::default()
+                    },
+                ));
+                sentry::integrations::panic::register_panic_handler();
+                sentry::integrations::env_logger::init(None, Default::default());
```
Comment (Kerollmops) [inline meilisearch-http/src/main.rs:41]: Are you sure that it works correctly? I remember that the scope in which you register the panic handler counts.

File: meilisearch-http/Cargo.toml [hunk 2]
```diff
@@ -32,17 +32,18 @@ mime = "0.3.16"
 pretty-bytes = "0.2.2"
 rand = "0.7.2"
 rayon = "1.2.0"
+sentry = { version = "0.18.0", features = ["with_rustls", "with_env_logger"] }
```
Comment (Kerollmops) [inline meilisearch-http/Cargo.toml:35]: The sentry dependency is really heavy in term of number of added dependencies, isn't there a way to reduce that? (765 additions, that's a lot)

File: README.md [hunk 3]
```diff
@@ -204,6 +204,7 @@ Hey! We're glad you're thinking about contributing to MeiliSearch! If you think
 
 Once a day, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
 Only information about the platform on which the server runs is stored. No other information is being sent.<br/>
+We also use Sentry to report us crashs or errors. To get more information about what iformation Sentry collect, please visit the [Sentry privacy policy](https://sentry.io/privacy/).
```
Comment (Kerollmops) [inline README.md:207]:
```suggestion
We also use Sentry to make us crash and error reports. If you want to know more about what Sentry collects, please visit their [privacy policy website](https://sentry.io/privacy/).
```

File: meilisearch-http/Cargo.toml (hunk 2 above)
Comment (qdequele) [reply meilisearch-http/Cargo.toml:35]: I haven't figured out which features to remove.

**Pull request closed without merging**